api/src/routes/status.ts

```diff
@@ -0,0 +1,8 @@
+import { Router } from "express"
+import { checkStatus } from "../controllers/status"
+
+const router: Router = Router()
+
+router.get("/", checkStatus)
```

Comment:
Can we add a test for this endpoint?

Reply:
added a basic integration test with supertest